source/about/staff.rst

```diff
@@ -53,16 +52,17 @@ The **Project Managers** of Sponge each coordinate a separate aspect of the Spon
 |                                   |                                    |
 |                                   | Website Manager                    |
 +-----------------------------------+------------------------------------+
+| phase                             | Ore Back-end Manager               |
++-----------------------------------+------------------------------------+
 | phroa                             | IRC Manager                        |
 +-----------------------------------+------------------------------------+
 | progwml6                          | Ore Back-end Manager               |
```

Comment:
nix the Ore Back-end Manager bit, since it's bundled with Systems Manager.

Reply:
`Ore Review Team Manager`